README.md

```diff
@@ -34,4 +34,4 @@ This data in addition to the other single-cell metadata is stored as a parquet f
 To understand each treatments ability to induce each of the 15 phenotypes, we perform post hoc testing with the probabilities between each of the treatments and the control (DMSO).
 These probabilities were generated in `2.evaluate_data`.
 In each of these tests we compute the median to determine the directionality of significance among treatments.
-We later use the Bonferonni correction to adjust the significance level in the visualization step of the anlaysis.
```

Comment:
Is there a reason you removed this? Is this not used as I did see your use Bernoulli adjustment in the file so would you reference that instead?

Reply:
Also, you might want to consider moving this into the main README for the module instead if this comment does still apply.